packages/amplify-graphql-api-construct-tests/src/__tests__/amplify-table-1.test.ts

```diff
@@ -4,12 +4,12 @@ import { cdkDestroy, initCDKProject, cdkDeploy, updateCDKAppWithTemplate } from
 
 jest.setTimeout(1000 * 60 * 60 /* 1 hour */);
 
-describe('CDK amplify table', () => {
+describe('CDK amplify table 1', () => {
```

Comment:
Is there a difference between 1 and 2 that'd make more sense to the reader? It's not clear how these test suites or apps are functionally different.

Reply:
The theme is same for both tests. The split is to avoid the test exceeding the 1 hour which requires a credential refresh. The number in description is to stay the same as file name. I use this pattern as we have e2e tests like `api_#`, `schema-iterative-#`

Reply:
Okay, I'm fine w/ it for now in that case.